pointcloud/src/main/scala/geotrellis/spark/pointcloud/triangulation/BoundaryDelaunay.scala

```diff
@@ -0,0 +1,287 @@
+package geotrellis.spark.pointcloud.triangulation
+
+import com.vividsolutions.jts.algorithm.distance.{DistanceToPoint, PointPairDistance}
+import com.vividsolutions.jts.geom.Coordinate
+
+import geotrellis.spark.buffer.Direction
+import geotrellis.spark.buffer.Direction._
+import geotrellis.vector.Extent
+import geotrellis.vector.triangulation.{DelaunayTriangulation, HalfEdgeTable, Predicates, TriangleMap}
+
+case class BoundaryDelaunay (dt: DelaunayTriangulation, boundingExtent: Extent) {
```

Comment:
Crap.  This will require carrying around the source `DelaunayTriangulation` object.  Will have to move construction over to the companion object.